Firestore/core/src/firebase/firestore/util/status.h

```diff
@@ -110,20 +116,37 @@ class ABSL_MUST_USE_RESULT Status {
  private:
   static const std::string& empty_string();
   struct State {
+    State() = default;
+    State(const State& other);
+
     FirestoreErrorCode code;
     std::string msg;
+    std::unique_ptr<PlatformError> wrapped;
```

Comment:
Nit: `wrapped` is a little generic. It also suggests that it's a `wrapped` _state_, which actually isn't the case. Maybe something like `underlying_error`, `platform_error`, `platform_specific_error`, `original_error`, `cause`, etc.?